lang/main.json

```diff
@@ -268,7 +272,8 @@
         "shareVideoLinkError": "Please provide a correct youtube link.",
         "removeSharedVideoTitle": "Remove shared video",
         "removeSharedVideoMsg": "Are you sure you would like to remove your shared video?",
-        "alreadySharedVideoMsg": "Another member is already sharing video. This conference allows only one shared video at a time.",
+        "alreadySharedVideoMsg": "Another member is already sharing a video. This conference allows only one shared video at a time.",
+        "alreadySharedVideoTitle": "Shared video already in the meeting",
```

Comment:
I'm not quite happy with this one. Maybe "Single shared video allowed" would be a better title. WDYT? Do you have any suggestions?

Reply:
"Single shared video allowed" is not bad. Maybe "Only one shared video is allowed at a time".

Reply:
I like the idea of stating the rule in the title.